jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/MultiPartFormDataTest.java

```diff
@@ -677,7 +675,7 @@ public void testPartWithBackSlashInFileName() throws Exception
 
         String contents = """
             --AaB03x\r
-            content-disposition: form-data; name="stuff"; filename="Taken on Aug 22 \\ 2012.jpg"\r
+            content-disposition: form-data; name="stuff"; filename="C:\\Pictures\\2012.jpg"\r
```

Comment:
We can't have a filename with spaces? 
If so we should keep the test and verify we cannot.

Reply:
The test is not about spaces. It is about how '\' is treated. 
I just made the name more realistic... but I'll put spaces back in.